### PR TITLE
Fix ramp election ep syncing and add syncing of claim review eps

### PR DESCRIPTION
**Pull request by kahlouie**
Connects #6186

### CODE REVIEW

File: spec/models/ramp_election_spec.rb [hunk 1]
```diff
@@ -82,20 +82,19 @@
...
       expect(ramp_election).to receive(:recreate_issues_from_contentions!)
-      expect(ramp_election).to receive(:sync_ep_status!)
 
       subject
     end
 
     context "when error is raised" do
       before do
-        expect(ramp_election).to receive(:sync_ep_status!).and_raise(ActiveRecord::RecordInvalid)
+        # TODO: What kind of error would be realistic to be raised here?
```
Comment (kahlouie) [inline spec/models/ramp_election_spec.rb]: Please note the question I have posed here. Wanted to make sure this is still the right way to check the rescue case, but also that we're checking with a potentially realistic error.

File: app/models/ramp_election.rb [hunk 2]
```diff
@@ -74,42 +74,33 @@ def rollback!
   end
 
   def sync!
-    create_end_product_establishment_if_missing
```
Comment (shanear) [inline app/models/ramp_election.rb]: confirmed this is no longer needed

File: app/jobs/ramp_election_sync_job.rb [hunk 3]
```diff
@@ -4,8 +4,7 @@ class RampElectionSyncJob < CaseflowJob
   application_attr :intake
 
   def perform(ramp_election_id)
-    RequestStore.store[:current_user] = User.system_user
-
-    RampElection.find(ramp_election_id).sync!
+    # TODO: DELETE ME
+    # we only didn't remove this to prevent a bunch of queued up job failures
```
Comment (shanear) [inline app/jobs/ramp_election_sync_job.rb]: well delete this in a follow up pr

File: app/models/ramp_election.rb [hunk 4]
```diff
@@ -49,9 +40,7 @@ def recreate_issues_from_contentions!
   end
 
   def successful_intake
-    @successful_intake ||= intakes.where(completion_status: "success")
-      .order(:completed_at)
-      .last
+    @successful_intake ||= intakes.where(completion_status: "success").order(:completed_at).last
```
Comment (shanear) [inline app/models/ramp_election.rb]: just changing a multi-liner to a single liner for 💅

File: app/models/ramp_closed_appeal.rb [hunk 5]
```diff
@@ -38,7 +38,7 @@ def reclose!
 
     # If the end product was canceled, don't re-close the VACOLS appeal.
     # Instead rollback the RAMP election data from Caseflow
-    return ramp_election.rollback! if ramp_election.end_product_canceled?
+    return ramp_election.rollback! if ramp_election.end_product_establishment.status_canceled?
```
[the remark this answers is not in the document]
Comment (kahlouie) [reply app/models/ramp_closed_appeal.rb]: I'll fix this in the next pr, so we can get this out sooner.

**Pull request merged**